### PR TITLE
Add integration tests

### PR DESCRIPTION
**Pull request by alexbourret**
Story details: https://app.shortcut.com/dataiku/story/110252

### CODE REVIEW

File: tests/python/unit/test_common.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import pytest
```
Comment (MayeulRousselet) [inline tests/python/unit/test_common.py:1]: What is the use of this file?

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]

File: tests/python/unit/test_common.py (hunk 1 above)
Comment (alexbourret) [reply tests/python/unit/test_common.py]: Tradition, Sir.
But ok then, removed in [80e9067](https://github.com/dataiku/dss-plugin-googlesheets/pull/4/commits/80e90679db6a7bffa644251884225fc6cce31628)

File: Makefile [hunk 2]
```diff
@@ -13,41 +13,45 @@ plugin:
 	@mkdir dist
 	@echo "{\"remote_url\":\"${remote_url}\",\"last_commit_id\":\"${last_commit_id}\"}" > release_info.json
 	@git archive -v -9 --format zip -o dist/${archive_file_name} HEAD
-	@zip --delete dist/${archive_file_name} "tests/*"
```
Comment (MayeulRousselet) [inline Makefile]: Should be kept, we don't want to distribute the tests  with the release

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]

File: tests/python/integration/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+pytest==6.2.1
+dataiku-api-client
+git+git://github.com/dataiku/dataiku-plugin-tests-utils.git@master#egg=dataiku-plugin-tests-utils
```
Comment (MayeulRousselet) [inline tests/python/integration/requirements.txt:3]: cf, https://github.com/dataiku/dss-plugin-template/pull/9
```suggestion
git+https://github.com/dataiku/dataiku-plugin-tests-utils.git@master#egg=dataiku-plugin-tests-utils
```

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]

File: tests/python/integration/requirements.txt (hunk 3 above)
Comment (MayeulRousselet) [reply tests/python/integration/requirements.txt:3]: git+git doesn't work anymore on localhost

**Review by MayeulRousselet — APPROVED** [review]
LGTM

**Pull request merged**